### PR TITLE
Add `cmd` extension for `npm` on Windows

### PR DESCRIPTION
**Pull request by garyb**

### CODE REVIEW

File: src/Pscid/Options.purs [hunk 1]
```diff
@@ -34,11 +34,14 @@ defaultOptions =
...
 
+npmCmd ∷ String
+npmCmd = if platform == Win32 then "npm.cmd" else "npm"
+
 mkDefaultOptions ∷ ∀ e. Eff (fs ∷ FS | e) PscidOptions
 mkDefaultOptions = do
   hbs ← hasBuildScript
   pure (defaultOptions {buildCommand = if hbs
-                                       then "npm run -s build"
+                                       then (pulpCmd <> " run -s build")
```
Comment (kritzcreek) [inline src/Pscid/Options.purs]: npmCmd? ^^

**Comment by garyb** [comment]
No idea what you're talking about :wink: :laughing:

**Comment by kritzcreek** [comment]
Thanks :+1:

**Comment by kritzcreek** [comment]
Released as 1.5.4

**Comment by garyb** [comment]
Thanks!

**Pull request merged**